static/js/components/material/Drawer.js

```diff
@@ -54,8 +62,8 @@ class Drawer extends React.Component {
     const { collections } = this.props;
     return <aside className="mdc-temporary-drawer mdc-typography" ref={div => this.drawerRoot = div}>
       <nav className="mdc-temporary-drawer__drawer">
-        <nav id="nav-username" className="mdc-temporary-drawer__content mdc-list">
-          <a className="mdc-list-item mdc-link" href="#">
+        <nav className="mdc-temporary-drawer__content mdc-list">
+          <a id="collapse_item"  className="mdc-list-item mdc-link" href="#">
```

Comment:
There are 2 spaces after the id attribute

Reply:
OK